tests/scripts/thread-cert/Cert_5_1_13_RouterReset.py

```diff
@@ -196,7 +196,7 @@ def verify(self, pv):
                        p.mle.tlv.addr16 is not nullField and \
                        p.mle.tlv.route64.id_mask is not nullField and\
                        p.mle.tlv.addr16 == ROUTER_RLOC16 and\
-                       p.sniff_timestamp - _pkt_lq.sniff_timestamp <= 1
+                       p.sniff_timestamp - _pkt_lq.sniff_timestamp < 1.1
```

Comment:
```suggestion
                       p.sniff_timestamp - _pkt_lq.sniff_timestamp < MLE_MAX_RESPONSE_DELAY + 0.1
```

Define `MLE_MAX_RESPONSE_DELAY` ?

Reply:
added